src/libraries/System.Net.Connections/src/System.Net.Connections.csproj

```diff
@@ -14,13 +15,18 @@
     <Compile Include="System\Net\Connections\ConnectionFactory.cs" />
     <Compile Include="System\Net\Connections\ConnectionListener.cs" />
     <Compile Include="System\Net\Connections\IConnectionProperties.cs" />
+    <Compile Include="System\Net\Connections\Sockets\SocketConnection.cs" />
+    <Compile Include="System\Net\Connections\Sockets\SocketsConnectionFactory.cs" />
+    <Compile Include="System\Net\Connections\Sockets\TaskSocketAsyncEventArgs.cs" />
   </ItemGroup>
   <ItemGroup>
     <Reference Include="System.Runtime" />
     <Reference Include="System.Memory" />
     <Reference Include="System.Net.Primitives" />
+    <Reference Include="System.Net.Sockets" />
     <Reference Include="System.Threading" />
     <Reference Include="System.Threading.Tasks" />
     <Reference Include="System.IO.Pipelines" />
+    <Reference Include="Microsoft.Win32.Primitives" />
```

Comment:
What is this needed for?

Reply:
It contains `Win32Exception` which is the subclass of `SocketException` and also somehow referenced by `NetworkException`. Nothing builds without it, IDK why isn't the reference transitive.